### DISCUSSION TITLE
Add nuspec parser

### DISCUSSION THREAD

**Discussion by patriksvensson** [cake-build/cake#3510] [Extension requests]
I'm currently helping out with a project that has a build script that parses nuspec files in one of it's build targets. I got it all working correctly but realized it would be nice to be able to parse the NuSpec into an object instead of driving myself crazy with XDocument.

**Comment by gep13** [comment]
@patriksvensson if we are doing this for nuget package nuspec files, would it make sense to also do this for chocolatey package nuspec files?  These are essentially the same thing, however, Chocolatey adds some additional elements into the nuspec file.

**Comment by patriksvensson** [comment]
@gep13 Sounds like another issue. Should probably only parse nuspec files as it is and add this later. Maybe just as an adapter to the NuSpec class.

**Reply by gep13** [reply]
Makes sense to me.
